conf/es/spain03.yml

```diff
@@ -463,6 +468,21 @@ years:
     names:
       en: Asunción
       es: Asunción
+   - public_holiday: true
+    date: '2021-08-16'
+    names:
+      en: Asunción in Lieu
+      es: Asunción in Lieu
+   - public_holiday: true
+    date: '2021-08-16'
```

Comment:
Won't this one (Toma de Málaga por los Reyes Católicos) need to have an in lieu day as well? Given that Asunción in lieu will fall on the 16th now?

Reply:
Ah yes, it should be moved to the 19th - is this something I correct or easier for you?

Reply:
OK, cool I can do it! :)